apps/web/src/lang/fr.json

```diff
@@ -10602,6 +10602,10 @@
     "defaultMessage": "Classification",
     "description": "Label displayed on the pool form classification field."
   },
+  "w0IA+c": {
+    "defaultMessage": "Ajouter une note",
```

Comment:
"Add notes" != "Ajouter une note"

Notes is plural in English, but then singular in the French translation. Did you mean: Ajouter des notes?

Reply:
Nice catch! I got it from a gov site, I guess I wanted to double check if it was correct and copied the wrong one.
Fixed by [e68d0f5](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10532/commits/e68d0f5cbf0d3c157061b2ce8c88ced40cd82430)